Core/clim-core/commands.lisp

```diff
@@ -223,6 +223,29 @@ designator) inherits menu items."
 
 (make-command-table 'user-command-table)
 
+<<<<<<< HEAD
```

Comment:
invalid cl, please rebase your changes against master branch instead of merging.